plugin.json

```diff
@@ -7,7 +7,7 @@
   "release_notes_url": "https://github.com/mattermost/mattermost-plugin-jira/releases/tag/v2.4.0",
   "icon_path": "assets/icon.svg",
   "version": "3.0.0-beta2.1",
-  "min_server_version": "5.24.0",
+  "min_server_version": "5.26.2",
```

Comment:
Why not `5.26.0`?

Reply:
It was a decision made in an impromptu meeting with @DHaussermann and @aaronrothschild.  What are your thoughts?